content/articles/2017-08-purrr-0.2.3.html

```diff
@@ -19,7 +19,7 @@
 <p>Install the latest version of purrr with:</p>
 <pre class="r"><code>install.packages(&quot;purrr&quot;)</code></pre>
 <p>You can read about the complete set of changes at <a href="https://github.com/tidyverse/purrr/releases/tag/v0.2.3" class="uri">https://github.com/tidyverse/purrr/releases/tag/v0.2.3</a>. In this blog post, I will present the highlights of this version which are a family of generic mappers, a new <code>pluck()</code> function for accessing deep elements in a readable way, and a few nifty features and improvements.</p>
-<div id="generic-mapping" class="section level3">
+<section id="generic-mapping" class="level3">
```

Comment:
Where do these changes come from?

Reply:
I don't know. I just rebuilt and committed the rendered html too.

Reply:
Let me try and reset these files so we can see whether the CI passes and the page looks as desired.

Reply:
Done.